cla_backend/libs/eligibility_calculator/cfe_civil/income.py

```diff
@@ -0,0 +1,37 @@
+from cla_backend.libs.eligibility_calculator.cfe_civil.conversions import pence_to_pounds
+
+
+def translate_income(income_data):
+    regular_transactions = []  # pension, maintenance_received, benefits
+
+    if hasattr(income_data, "tax_credits") and income_data.tax_credits > 0:
```

Comment:
As previously commented, would be good to push the check for the field into the method so that we are confident of not making a typo due to duplication